browser/src/control/Control.NotebookbarImpress.js

```diff
@@ -423,10 +423,32 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 				'command': _('Toggle UI Mode')
 			},
 			{
-				'id': 'showstatusbar',
-				'type': 'menubartoolitem',
-				'text': _('Status Bar'),
-				'command': _('Show Status Bar')
+				'type': 'container',
+				'children': [
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'showstatusbar',
+								'type': 'menubartoolitem',
+								'text': _('Status Bar'),
+								'command': _('Show Status Bar')
+							}
+						]
+					},
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'collapsenotebookbar',
+								'type': 'menubartoolitem',
+								'text': _('Tabbed Bar'),
+								'command': _('Collapse Notebook Bar')
+							}
+						]
+					}
```

Comment:
Can you please add the new button before the status bar button? (so is becomes near to the place it affects)